index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Birthday Card Generator</title>
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.0/css/bulma.min.css">
+        <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,600;1,200&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=K2D:wght@700&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Averia+Serif+Libre&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Nixie+One&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Abril+Fatface&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="css/style.css">
+    </head>
+    <body>
+        <header class="hero">
+            <div class="hero-body">
+                <div class="container">
+                    <h1 class="page-title center"> Birthday Card Maker</h1>
+                </div>
+            </div>
+        </header>
+        <main>
+            <div class="section">
+                <div class="container input-container">
+                    <section class="section">
+                        <div class="container">
+                            <div class="columns">
+                                <div class="column is-half is-offset-one-quarter">
+                                    <div class="content">
+                                        <h1>Enter Name:</h1>
+                                        <div>
+                                            <input id="input-name" class="input is-danger" type="text" placeholder="Name">
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </section>
+                    <section class="section center">
+                        <div class="container">
+                            <div class="columns">
+                                <div class="column is-half is-offset-one-quarter choose-bday">
+                                    <div class="content">
+                                        <h1>Choose Birthday:</h1>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="columns">
+                                <div class="column is-2 is-offset-one-quarter">
+                                    <div class="select is-info" style="width: 100%">
```

Comment:
It's a best practice to avoid in-line styles if at all possible. CSS should be kept to CSS files or style tags.